### PR TITLE
[DEV-6970] Federal Accounts Sankey Copy Edit:

### PR DESCRIPTION
**Pull request by MaxwellKendall**
Copy edit

- [x] Code review complete

### CODE REVIEW

**Review by ebdabbs — CHANGES_REQUESTED** [review]
Screen Shot 2021-03-22 at 10 05 51 AM
could we also change the mobile Sankey label?

**Comment by MaxwellKendall** [comment]
@ebdabbs ayeeee good catch

File: src/js/components/account/visualizations/sankey/SankeyVisualizationVertical.jsx [hunk 1]
```diff
@@ -453,7 +453,7 @@ obligations incurred`}
                             endX={this.state.bottom.unobligated.x + this.state.bottom.flow.x}
                             width={this.state.bottom.unobligated.width}
                             length={this.state.bottom.flow.length}
-                            description={`Flow of money out of total budget authority to \
+                            description={`Flow of money out of total budgetary resources to \
```
Comment (ebdabbs) [inline src/js/components/account/visualizations/sankey/SankeyVisualizationVertical.jsx:456]: Thanks for updating these fields too!

**Review by ebdabbs — APPROVED** [review]

**Comment by badassconnection** [comment]
﻿Thanks you can deposit on my account for  donation for good  service  td Canada trust my name jimmy phone 514 797 0513 

Badassconnection a votre service

**Comment by badassconnection** [comment]
﻿
﻿Thanks you can deposit on my account for  donation for good  service  td Canada trust my name jimmy phone 514 797 0513 

Badassconnection a votre service

**Pull request merged**